docs/safemode.md

```diff
@@ -88,4 +88,10 @@ spec:
     path: /mnt/data # mount point (in the pod) to apply throttle on
     throttling:
       readBytesPerSec: 1024 # read throttling in bytes per sec
-```
\ No newline at end of file
+```
+
+## FAQ
+
+### Why is the namespace/cluster threshold not equal to what I specified in my Disruption?
+It may be the case that 0 was used as namespace/cluster threshold. Because 0 has no value as a threshold, it is simply
```

Comment:
Do we care about the 0 value? Like, would it be useful for someone to want to specify 0 in the disruption?
If yes, why don't we turn this into a pointer to distinguish 0 from not specified?

Reply:
0 is not useful. 0 threshold means "Trigger Safemode if the target count is > 0%" which will almost always be the case and really means nothing. We are assuming 0 is a mistake if specified or unset, in bot situations defaults to the cluster specified thresholds.